### DISCUSSION TITLE
Materials list Export / Print

### DISCUSSION THREAD

**Discussion by Letech01** [Drumstix42/drumstix42.github.io#5] [Ideas]
Hi ,

Nice work, 
ideas : 
1- Having a Export + Print Button on each TAB so you can export the list to a CSV file..
2- Button that would create a to do list with  the material list from one or all the TAB..

**Comment by Drumstix42** [comment]
Thanks for your suggestions. I will consider implemented them when I have time to work on some improvements to the tool. I will post any updates here as I get to them.
